examples/crewai/python/src/honcho_crewai/storage.py

```
@@ -0,0 +1,207 @@
+
+            content_str = str(value)
+
+            # Add message to session
+            self.session.add_messages([peer.message(content_str, metadata=metadata)])
+
+            logger.debug(
+                f"Saved message from {metadata.get('name', role)}: {content_str[:100]}..."
+            )
+
+        except Exception as e:
+            logger.error(f"Error saving to Honcho: {e}")
```

Ruff (0.14.5)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)